### ISSUE TITLE
Collections page not displaying correctly

### ISSUE THREAD

**Issue by Alwurts** [mmistakes/minimal-mistakes#1752]
## Description
  Hi, I have been trying to use the collections feature of this them; I followed the documentation for it and created the portfolio example, however when i follow then link to the portfolio collection the page displayed shows the document its supposed to have but it has no format.
Attached below is a screen of the issue i get.
![image](https://user-images.githubusercontent.com/32560488/43030857-184be012-8cc9-11e8-975c-60c98a9b8b34.png)

## Environment

- Minimal Mistakes version: 4.12 - I think its the latest, i just started using it.
- Ruby gem or remote theme version:  2.5.0
- Jekyll version: jekyll 3.5

---

## Expected behavior

I was expecting to get a collection page such as pets or portfolio as it is in the current example in the documents.
## Steps to reproduce the behavior

Basically i followed the steps in the documentation, in order to set the portfolio collection working.

**Comment by mmistakes** [comment]
I think you're missing the portfolio collection archive page. [See this example](https://github.com/mmistakes/minimal-mistakes/blob/master/docs/_pages/portfolio-archive.md).

**Comment by Alwurts** [comment]
Yes you are correct.
That completely solved the problem.

Thank you very much for your hard work.
